diff --git a/dist/commonjs/index.js b/dist/commonjs/index.js
index c12dc5e6476874f9dbba429a19e445a4a688e1df..a3f2bc19dd37aedd8fb893f02ef58a3fa5c613e7 100644
--- a/dist/commonjs/index.js
+++ b/dist/commonjs/index.js
@@ -4,7 +4,7 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
 };
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.unescape = exports.escape = exports.AST = exports.Minimatch = exports.match = exports.makeRe = exports.braceExpand = exports.defaults = exports.filter = exports.GLOBSTAR = exports.sep = exports.minimatch = void 0;
-const brace_expansion_1 = __importDefault(require("brace-expansion"));
+const brace_expansion_1 = require("brace-expansion");
 const assert_valid_pattern_js_1 = require("./assert-valid-pattern.js");
 const ast_js_1 = require("./ast.js");
 const escape_js_1 = require("./escape.js");
@@ -157,7 +157,7 @@ const braceExpand = (pattern, options = {}) => {
         // shortcut. no need to expand.
         return [pattern];
     }
-    return (0, brace_expansion_1.default)(pattern);
+    return (0, brace_expansion_1.expand || brace_expansion_1.default || brace_expansion_1)(pattern);
 };
 exports.braceExpand = braceExpand;
 exports.minimatch.braceExpand = exports.braceExpand;
diff --git a/dist/esm/index.js b/dist/esm/index.js
index 737c8095415235e69e4717b505d0ee4ea3c0b6fa..ad5aa644f7957fb39627327bdd6435f4007dca30 100644
--- a/dist/esm/index.js
+++ b/dist/esm/index.js
@@ -1,4 +1,4 @@
-import expand from 'brace-expansion';
+import { expand } from 'brace-expansion';
 import { assertValidPattern } from './assert-valid-pattern.js';
 import { AST } from './ast.js';
 import { escape } from './escape.js';
