diff --git a/minimatch.js b/minimatch.js
index 8545b3ab506a3185107a8f24a4c150a7b7b585f8..7c5de40101bb0a85ce5de9f8387d51710021c89d 100644
--- a/minimatch.js
+++ b/minimatch.js
@@ -16,7 +16,8 @@ minimatch.sep = path.sep
 
 const GLOBSTAR = Symbol('globstar **')
 minimatch.GLOBSTAR = GLOBSTAR
-const expand = require('brace-expansion')
+const braceExpansion = require('brace-expansion')
+const expand = braceExpansion.expand || braceExpansion
 
 const plTypes = {
   '!': { open: '(?:(?!(?:', close: '))[^/]*?)'},
